bin/pycbc_inspiral

```diff
@@ -25,6 +25,7 @@ from pycbc.types import TimeSeries, FrequencySeries, zeros, float32, complex64
 import pycbc.fft.fftw, pycbc.version
 import pycbc.opt
 import pycbc.weave
+import pycbc.injcutter
```

Comment:
We need a human-readable name for the option to the code - see my previous comment that this should consist of whole English words separated by dashes - and maybe an internal name for the module. 
My best recommendation for the option so far was `--injection-skip-templates`